doxia-modules/doxia-module-markdown/src/test/java/org/apache/maven/doxia/module/markdown/MarkdownSinkTest.java

```diff
@@ -349,11 +350,34 @@ public void testMultipleAuthors() {
                 + "author: " + EOL
                 + "  - first author" + EOL
                 + "  - second author" + EOL
-                + MarkdownMarkup.METADATA_MARKUP + EOL;
+                + MarkdownMarkup.METADATA_MARKUP + EOL + EOL;
 
         assertEquals(expected, getSinkContent(), "Wrong metadata section");
     }
 
+    @Test
+    public void testRoundtrip() throws IOException, ParseException {
+        parseFile(parser, "test", getSink());
+
+        final SinkEventTestingSink regeneratedSink = new SinkEventTestingSink();
+        try (Reader reader = new StringReader(getSinkContent())) {
+            parser.parse(reader, regeneratedSink);
+        }
+
+        System.out.println(getSinkContent());
```

Comment:
delete this line; passing tests should generate no output

Reply:
Done in https://github.com/apache/maven-doxia/pull/146/commits/c49775264627b41daa3b17e90f9d72a0b0d9c218.